_articles/2021-05-12-test-article-1.md

```diff
@@ -6,6 +6,7 @@ categories:
   - article
   - test
 last_modified_at: 2021-06-24T08:05:34-05:00
+image: /assets/images/page-header-og-image.png
```

Comment:
Has this asset been committed?  I can't see it anywhere.

Reply:
this is test image string for seo og tag feature, i have not added actual image in assets dir.